ci/cases/yamls/atmaerosnowDA_defaults_ci.yaml

```diff
@@ -3,3 +3,4 @@ defaults:
 base:
   DO_JEDISNOWDA: "YES"
   ACCOUNT: {{ 'HPC_ACCOUNT' | getenv }}
+  DO_TEST_MODE: "YES"
```

Comment:
Since I plan on eventually adding ```DO_JEDI_TESTING: "YES"``` here as well once we have test references for JEDI aero and snow DA, would it make sense to change this to ```DO_GSI_TESTING``` or something like that?

Reply:
Good question, are there other potential uses of the testing flag beyond DA? Asking g-w code maintainers for their input.

Reply:
Alternatively, ```DO_TEST_MODE``` could turn on testing mode for both GSI and JEDI

Reply:
I think I would prefer a single flag to indicate testing mode.  It's possible we could find a good use for a test mode with the forecast as well (perhaps shortening forecasts to 48 hours).  We could then run with `DO_TEST_MODE=NO` for weekly tests to capture the full range of the analyses and forecasts.

Reply:
I agree. I'll update GW PR [#3120](https://github.com/NOAA-EMC/global-workflow/pull/3120) to use this flag as well.